### PR TITLE
linting via our config

### PR DESCRIPTION
**Pull request by purplecabbage**
Some basic cleanup, and switched to use our eslint config

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -2,9 +2,9 @@
   "name": "@adobe/aio-lib-core-tvm",
   "version": "1.3.0",
   "description": "A JS client to access the Adobe I/O token vending machine",
-  "main": "index.js",
+  "main": "lib/TvmClient.js",
```
Comment (moritzraho) [inline package.json:5]: This is not equivalent to the index.js and breaks the interface

=> index returns `require('lib/TvmClient.js').TvmClient` while this returns { TvmClient }

**Review by moritzraho — CHANGES_REQUESTED** [review]
revert the index.js deletion and package.json main change, otherwise lgtm :)

File: package.json (hunk 1 above)
Comment (Himavanth) [reply package.json:5]: Wouldn't the change made to `module.exports` in TvmClient.js take care of this ?

**Review by moritzraho — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (moritzraho) [reply package.json:5]: yup I went too fast, thanks :)

**Pull request merged**